build.zig

```diff
@@ -1,45 +1,44 @@
 const std = @import("std");
+const FileSource = @import("std").build.FileSource;
 
 pub fn build(b: *std.Build) !void {
-	const target = b.standardTargetOptions(.{});
-	const optimize = b.standardOptimizeOption(.{});
+    const target = b.standardTargetOptions(.{});
+    const optimize = b.standardOptimizeOption(.{});
 
-	const smtp_module = b.addModule("smtp_client", .{
-		.root_source_file = .{ .path = "src/smtp.zig" },
-	});
+    const smtp_client = b.addModule("smtp_client", .{ .source_file = FileSource.relative("src/smtp.zig") });
```

Comment:
This is no longer valid...it was changed in master a week or two agao

I'm fine with a Zig 0.11 compatible version,but it should be in a "zig-0.11" branch.

Reply:
Thank you